iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/sql/ast/LoadTsFile.java

```diff
@@ -232,6 +232,43 @@ public boolean reconstructStatementIfMiniFileConverted(final List<Boolean> isMin
     return tsFiles == null || tsFiles.isEmpty();
   }
 
+  /**
+   * Splits the current LoadTsFile statement into multiple sub-statements, each handling one TsFile.
+   * Used to support batch execution when loading multiple files.
+   *
+   * @return the list of sub-statements
+   */
```

Comment:
Is it possible that each sub-statement handles n (maybe configurable) TsFiles?
Will n affect performance? May need an experiment.